notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -28,10 +28,10 @@
     "- [Select model for inference](#Select-model-for-inference)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">2 + 2 = 3 
</pre><p>Oops :) Seems like 60% is too high. How about make it 50%? For it, the answer is correct. I'll post results to the ticket.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1492/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>